Jenkinsfile

```diff
@@ -53,6 +53,13 @@ pipeline {
                     ln -s $HOME/data/testing/${env.JOB_NAME}/sample0/output sample0
                     echo "[ Run test now ]"
                     pytest -s
+                    echo "[ Linking test data ]"
+                    rm -rf sample0
+                    mkdir sample0
+                    ln -s $HOME/data/testing/${env.JOB_NAME}/sample0/input sample0
+                    ln -s $HOME/data/testing/${env.JOB_NAME}/sample0/output sample0
```

Comment:
Why do you nuke the test data for the MPI test?

Reply:
The test data gets nuked after running pytest -s
Then need to recreate sample0 and relink again to run the mpi test

Reply:
:( If we aren't downloading the data in python, we shouldn't be deleting it either... 
This is fine for now though